unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs

```diff
@@ -70,9 +72,13 @@ internal void UpdateLODsBillboard()
         {
             foreach (var kvp in lodControllers)
             {
-                otherPlayers.TryGetValue(kvp.Key, out Player player);
+                Player player = kvp.Value.player;
+
+                if (!IsInFrontOfCamera(player.worldPosition))
+                    continue;
+
                 Vector3 previousForward = player.forwardDirection;
-                Vector3 lookAtDir = (player.worldPosition - CommonScriptableObjects.cameraPosition).normalized;
+                Vector3 lookAtDir = (player.worldPosition - cameraPosition.Get()).normalized;
```

Comment:
This `Get()` calls are expensive, we can cache it before the loop.

Reply:
You are right, I cached the `cameraPosition` and `cameraForward` so that it's only updated once during the `Update()`